src/UI/templates/js/Core/src/core.Tooltip.js

```diff
@@ -12,7 +12,7 @@ class Tooltip {
      * The container of the tooltip and the trigger element.
      * @type {Element}
      */
-    #container:
```

Comment:
@thibsy / @lzehnder please revert those changes since these are not part of this refactoring